nima_core/precognition/storage.py

```
@@ -625,4 +626,95 @@ def get_recent(self, limit: int = 5) -> List[Dict[str, Any]]:
+    def dismiss(self, item_id: str) -> bool:
+        """Mark a follow-up item as dismissed — returns True if found, False if not."""
+        conn = self._conn()
+        try:
+            cursor = conn.execute("UPDATE follow_up_items SET dismissed = 1 WHERE id = ?", (item_id,))
+            conn.commit()
+            return cursor.rowcount == 1
+        finally:
+            conn.close()
+
+
+__all__ = ["PrecognitionStorage", "FollowUpStore"]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)